xbmc/pvr/addons/PVRClientMenuHooks.cpp

```diff
@@ -26,7 +26,7 @@ CPVRClientMenuHook::CPVRClientMenuHook(const std::string& addonId, const PVR_MEN
       hook.category != PVR_MENUHOOK_TIMER &&
       hook.category != PVR_MENUHOOK_EPG &&
       hook.category != PVR_MENUHOOK_RECORDING &&
-      hook.category != PVR_MENUHOOK_DELETED_RECORDING &&
+      hook.category != PVR_MENUHOOK_RECORDING &&
```

Comment:
rebase was successful, but this change is still wrong.
please now
`git checkout HEAD^ -- xbmc/pvr/addons/PVRClientMenuHooks.cpp`
`git commit --amend --no-edit`
`git push -f`

Reply:
@howie-f I did it. 
![image](https://user-images.githubusercontent.com/45079270/107820384-15c5df80-6d51-11eb-9427-4749ffc937a1.png)

Reply:
i'm sorry, `git commit -a --amend --no-edit` is correct.
i missed to write -a

Reply:
You're right, sorry. 
I missed to write -a.

Reply:
sorry, but the change is still there...
please once again the three commands in a row.. it should work

Reply:
I continue having four files changed.
![image](https://user-images.githubusercontent.com/45079270/107821710-5161a900-6d53-11eb-941d-8aaf00bef8c0.png)

Reply:
tried it on your repo:
```
fhw:~/domi/xbmc (fix-self-assigment-bug) $ git checkout HEAD^ -- xbmc/pvr/addons/PVRClientMenuHooks.cpp
fhw:~/domi/xbmc (fix-self-assigment-bug) $ git commit -a --amend --no-edit
[fix-self-assigment-bug 637ef3a5bf] [fix] remove self-assigment
 Author: Elias Rodriguez <45079270+domielias@users.noreply.github.com>
 Date: Wed Nov 4 14:25:17 2020 -0400
 3 files changed, 3 insertions(+), 4 deletions(-)
```
can it be that you need to `cd D:\Open Source Project\PullRequest` before you issue the first command?

Reply:
maybe windows cannot understand HEAD^, please try this instead
`git checkout  4a937d9 -- xbmc/pvr/addons/PVRClientMenuHooks.cpp`
`git commit -a --amend --no-edit`

Reply:
Yes, Windows don't recognize the HEAD. That work

Reply:
juhuuuu, then push -f

Reply:
I did it.